content/blog/2019/custom-ingress-gateway/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Deploy a custom ingress gateway using cert-manager
+description: Describes how to deploy a custom ingress gateway using cert-manager manually.
+subtitle: Custom ingress gateway
+publishdate: 2019-01-08
+weight: 89
+keywords: [ingress,traffic-management]
+attribution: Julien Senon
+---
+
+This post provides instructions to manually create a custom ingress [gateway](/docs/reference/config/istio.networking.v1alpha3/#Gateway) with automatic provisioning of certificates based on cert-manager.
```

Comment:
Would be great if you can highlight why users would need to create a custom ingress gateway vs create an ingress gateway resource using the default ingress gateway provided by istio?   for example https://istio.io/docs/tasks/traffic-management/secure-ingress/